README.md

```diff
@@ -1,286 +1,116 @@
 # React components for TalkJS
 
-This library makes it easy to use the [TalkJS](https://talkjs.com) pre-built chat UIs inside a React web application.
+This library makes it easy to use [TalkJS](https://talkjs.com) pre-built chat UIs inside a React web application.
 
-`@talkjs/react` encapsulates `talkjs`, the framework-independent ("vanilla") [TalkJS JavaScript SDK](https://talkjs.com/docs/Reference/JavaScript_Chat_SDK). It only provides React components for UI-related matters: For anything related to data manipulation, such as synchronizing user data, or creating and joining conversations, use the vanilla SDK.
+`@talkjs/react` encapsulates `talkjs`, the framework-independent [TalkJS JavaScript SDK](https://talkjs.com/docs/Reference/JavaScript_Chat_SDK). It provides React components for UI-related matters. For anything related to data manipulation, such as synchronizing user data, or creating and joining conversations, use the regular SDK.
```

Comment:
Perhaps here at the end "... use the regular JavaScript SDK", again just to have it super explicit?

Reply:
Good idea, actually switched 'regular' -> 'JavaScript' as I think that's even better https://github.com/talkjs/talkjs-react/pull/5/commits/4d8aa57813ea6fde444d39e2606fc3fef3f9e9c1